cellmlmanip/model.py

```diff
@@ -401,12 +401,11 @@ def get_ontology_terms_by_symbol(self, symbol, namespace_uri=None):
         Will return a list of term names.
         """
         ontology_terms = []
-        cmeta_id = self.graph.nodes[symbol].get('cmeta_id', None)
-        if cmeta_id:
+        if symbol.cmeta_id:
             predicate = ('http://biomodels.net/biology-qualifiers/', 'is')
             predicate = create_rdf_node(predicate)
             for delimeter in ('#', '/'):  # Look for terms using either possible namespace delimiter
-                subject = rdflib.term.URIRef(delimeter + cmeta_id)
+                subject = rdflib.term.URIRef(delimeter + symbol.cmeta_id)
```

Comment:
Ah, not something you introduced, but this should always be `'#' + symbol.cmeta_id`!

Reply:
Are you absolutely sure about that? I distinctly remember `/` being possible too.

Reply:
(You can have # or / between NS URI and local part, but not as a pure prefix for a within-document id.)